src/Wifi.c

```diff
@@ -0,0 +1,67 @@
+#include "Wifi.h"
+
+#define SSID "Jesus"
+#define PWD "8715387153"
+
+bool s_isConnectionFault = false;
+
+void Wifi_connect() {
+    sdk_wifi_set_opmode(STATION_MODE);
+    sdk_wifi_station_set_auto_connect(1);
+    struct sdk_station_config config = {
+        .ssid = SSID,
+        .password = PWD,
+    };
+    sdk_wifi_station_set_config(&config);
+    // printf("SSID: %s, PWD: %s", config.ssid, config.password);
+}
+
+void onConnFault() {
+    s_isConnectionFault = true;
+    printf("Connection aborted\n");
+}
+
+void Wifi_checkConnection(stationStatus) {
```

Comment:
I don't see when this function checks the status. I purpose that this function must names as Wifi_eventHandler(uint8_t event) or Wifi_statusHandler(uint8_t  status). Also, need indicate type of variables.